### ISSUE TITLE
How to test RepMet on COCO or Pascal as regular detection?

### ISSUE THREAD

**Issue by duynn912** [jshtok/RepMet#25]
Hi @jshtok,

As in your paper, RepMet only provides a way to test on novel classes following episodes. After each episode, new novel classes will replace the old one.  Therefore, I have a problem if there is any way to test RepMet on benchmarks such as PASCAL VOC or COCO as regular detection for base and novel classes because I have two problems as follows:
(1) Assuming that I follow to train RepMet on PASCAL VOC that 15 classes for base classes and the rest of 5 classes is for novel classes or on COCO, 20 classes overlapped with PASCAL as novel classes and the rest is base classes. A problem here is how I can test RepMet on each image of test set. In ImageNetLOC, there is only one category with one or more instances in an image but with COCO or PASCAL dataset, we may have many categories including base and novel categories in an image.

(2) Could you please show me a way to get the result with the base classes of ImagaNetLOC in Table 4 (I have tried to do it but mAP just got 0%) and the results of 100 classes of ImageNet-LOC with no episode fine-tuning in Table 3?

I look forward to hearing from you. Hope you can help me out!

Best regards,
Duynn
